### ISSUE TITLE
Request for azure cli

### ISSUE THREAD

**Issue by wesleycoder** [webinstall/webi-installer-requests#80]
https://learn.microsoft.com/en-us/cli/azure/install-azure-cli

Always some pain to find the correct link/method to install on different systems.
Having this on webi would be awesome.

**Comment by coolaj86** [comment]
I only see the MSI in their GitHub Releases <https://github.com/Azure/azure-cli/releases/latest>.

Do you know if there is a self-contained binary for macOS or Linux that can be installed?

Or - just skimming the docs and noticing python references - is this a python script that could be installed with `pip`?

I need more information about how we might support this. Right now it looks like maybe we could give it a similar treatment to Rust where we don't _really_ support it (versioned symlinks et al), but we alias the install scripts. What do you think about that?

**Comment by wesleycoder** [comment]
Yeah, from my experience the setup on mac and linux is made through `pip` for a long time.
I believe the starting point is going through this:
https://github.com/Azure/azure-cli/blob/dev/scripts/install_full.sh
